src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "The ball has " + this.color + " color and number " + this.number;
+    }
```

Comment:
we use `this` when we need to differentiate between a class field and an argument that share the same name, it's not the case here, so you can safely omit them